documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -216,7 +216,7 @@ endif::[]
 
 [start=2]
 . Create a `Provider` manifest for the source provider:
-+
+
```

Comment:
I noticed a "+" being visible in the CLI procedure for migrating using a VMware source provider, so i removed it.